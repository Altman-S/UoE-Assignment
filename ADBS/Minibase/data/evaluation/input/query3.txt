Q(x,y) :- R(x, y, z)