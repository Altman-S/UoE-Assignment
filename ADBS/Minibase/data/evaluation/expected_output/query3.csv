1, 9
2, 7
4, 2
5, 0
8, 1
8, 9